solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -430,14 +492,23 @@ pub mod solana_ibc {
             .map_err(|err| error!((&err)))
     }
 
-    #[allow(unused_variables)]
+    /// The port_id, channel_id and hashed_full_denom are passed
```

Comment:
port_id and channel_id are no longer passed.

Reply:
right